lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { shopIdAutoValue } from "./helpers";
+import { Address } from "./address";
+import { Metafield } from "./metafield";
+
+/**
+ * Accounts Schemas
+ */
+
+export const Profile = new SimpleSchema({
+  addressBook: {
+    type: [Address],
+    optional: true
+  },
+  name: {
```

Comment:
Let's put a comment in somewhere to indicate what field we have eliminated? (I assume that's what you did here?)